index.html

```diff
@@ -0,0 +1,282 @@
+
+      </div>
+    </div>
+  </section>
+
+
+  <!-- portfolio section  -->
+  <section class="portfolio" id="portfolio">
+    <div class="main-text">
+      <h2><span>latest</span>Project</h2>
+    </div>
+    <div class="portfolio-content">
+      <div class="row">
+        <img src="img/project1.jpg">
+        <div class="layer">
+          <h5>Visual Design</h5>
+          <p>
+            Check Out 10 best Design s update for the top web design and development companies
+          </p>
+          <a href="https://farwamuhibzada.github.io/Technical_documentation_page/">
+            <i class="fa-solid fa-arrow-up-right-from-square"></i>
+          </a>
+        </div>
+      </div>
+
+      <div class="row">
+        <img src="img/project2.jpg">
+        <div class="layer">
+          <h5>Visual Design</h5>
+          <p>
+            Check Out 10 best Design s update for the top web design and development companies
+          </p>
+          <a href="https://farwamuhibzada.github.io/Product_Landing_page/">
+            <i class="fa-solid fa-arrow-up-right-from-square"></i>
+          </a>
+        </div>
+      </div>
+
+      <div class="row">
+        <img src="img/project3.jpg">
+        <div class="layer">
+          <h5>Visual Design</h5>
+          <p>
+            Check Out 10 best Design s update for the top web design and development companies
+          </p>
+          <a href="https://farwamuhibzada.github.io/Survey_Form/">
+            <i class="fa-solid fa-arrow-up-right-from-square"></i>
+          </a>
+        </div>
+      </div>
+
+      <div class="row">
+        <img src="img/project4.jpg">
+        <div class="layer">
+          <h5>Visual Design</h5>
+          <p>
+            Check Out 10 best Design s update for the top web design and development companies
+          </p>
+          <a href="https://farwamuhibzada.github.io/Personal_Portfolio_Webpage/">
+            <i class="fa-solid fa-arrow-up-right-from-square"></i>
+          </a>
+        </div>
+      </div>
+
+
+      <div class="row">
+        <img src="img/project5.jpg">
+        <div class="layer">
+          <h5>Visual Design</h5>
+          <p>
+            Check Out 10 best Design s update for the top web design and development companies
+          </p>
+          <a href="https://farwamuhibzada.github.io/Personal_Portfolio_Webpage/">
+            <i class="fa-solid fa-arrow-up-right-from-square"></i>
+          </a>
+        </div>
+      </div>
+
+      <div class="row">
+        <img src="img/project6.jpg">
+        <div class="layer">
+          <h5>Visual Design</h5>
+          <p>
+            Check Out 10 best Design s update for the top web design and development companies
+          </p>
+          <a href="https://farwamuhibzada.github.io/Technical_documentation_page/">
+            <i class="fa-solid fa-arrow-up-right-from-square"></i>
+          </a>
+        </div>
+      </div>
+    </div>
+
+  </section>
```

Comment:
- Remove this section as it is not useful for now:

![Screenshot 2024-06-06 210244](https://github.com/FarwaMuhibZada/Personal_Portfolio/assets/96838030/9ab08dbd-f8dc-41e4-9b63-bb35309a82be)